### DISCUSSION TITLE
Play character animation in code ?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [FyroxEngine/Fyrox#571] [Q&A] [answered]
Hi,

How do you play an animated FBX (character) in code ?
The book chapters cover the creation of animations but not the use of already existing ones (in code).

Thanks

**Comment by mrDIMAS** [comment, marked as answer]
Hi, is a shame that I didn't add this information before. I fixed this today - https://fyrox-book.github.io/fyrox/animation/animation.html#importing . Let me know if it helped or maybe something should be improved/added.

**Reply by ThomasCartier** [reply]
You rock :) Thanks for the complement, your engine is doing great here :+1:
